sorc/sfc_climo_gen.fd/interp.F90

```diff
@@ -1,13 +1,12 @@
 !> @file
-!! @author gayno @date 2018
+!! @brief Read the input source data and interpolate it to the
+!! model grid.
 !!
-!! Read the input source data and interpolate it to the
-!! model grid. 
+!! @author gayno @date 2018
 !!
```

Comment:
In this case we do need a blank line here. There are two documentation blocks here, one for the file, and one for the subroutine interp

Reply:
Done